.eslintrc.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
       files: ['*.stories.jsx'],
       rules: {
         'react/prop-types': 'off',
+        'react/no-multi-comp': 'off',
```

Comment:
What does this fix?

Reply:
This is an eslint error that was thrown in a bunch of `.stories` files because it didn't like multiple components being exported, which is what stories need